crates/uv-python/src/installation.rs

```diff
@@ -176,6 +177,14 @@ impl PythonInstallation {
         LenientImplementationName::from(self.interpreter.implementation_name())
     }
 
+    /// Whether this is a CPython installation or an alternative Python implementation.
```

Comment:
Can we rephrase to make clear that this returns `false` for CPython, `true` for everything else? As written it sounds like it returns `true` when the installation is CPython or any other implementation.